AI TEST - multi files [TESTING, DO NOT MERGE] (#44)
AdobeDocs/adp-devsite-github-actions-test: src/pages/ai-test/faq.md

> ---
> keywords:
>   - App Builder
>   - Extensibility
>   - API Documentation
>   - Developer Tooling
> title: Frequently Asked Questions
> ---

Author: ```suggestion
---
title: Frequently Asked Questions
description: This document provides answers to common questions about Adobe App Builder, including its features, usage, licensing, integration capabilities, technical details on Adobe I/O Runtime, and support resources.
keywords:
- App Builder
- Extensibility
- API Documentation
- Developer Tooling
- Adobe Experience Cloud
---
```

> ---
> keywords:
>   - App Builder
>   - Extensibility
>   - API Documentation
>   - Developer Tooling
> title: Frequently Asked Questions
> ---

Author: ```suggestion
---
title: Frequently Asked Questions
description: This document provides answers to common questions about App Builder, including its capabilities, licensing, extensibility, supported technologies, integration options, and Adobe I/O Runtime details.
keywords:
- App Builder
- Adobe Experience Cloud
- Licensing
- Extensibility
- I/O Runtime
other original metadata
---
```

> ---
> keywords:
>   - App Builder
>   - Extensibility
>   - API Documentation
>   - Developer Tooling
> title: Frequently Asked Questions
> ---

Reviewer: ```suggestion
---
title: Frequently Asked Questions
description: This document provides answers to frequently asked questions about App Builder, including its features, licensing, extensibility, usage, integration, and technical details related to Adobe Experience Cloud and Adobe Experience Platform.
keywords:
- App Builder
- Adobe Experience Cloud
- Licensing
- Extensibility
- I/O Runtime
- Developer Tooling
- API Documentation
---
```
